dz-6/src/main/java/org/example/Methods.java

```diff
@@ -0,0 +1,96 @@
+package org.example;
+
+import java.util.*;
+
+public class Methods {
+    public static void scanArray(ArrayList words) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter word:");
+        for (int i = 0; i < 4; i++) {
+            words.add(scanner.next());
+        }
+    }
+
+    public static ArrayList findUniqueChars(ArrayList words) {
+        ArrayList<String> uniqueList = new ArrayList<>();
+        ArrayList<Character> charList = new ArrayList<>();
+
+        for (int i = 0; i < words.size(); i++) {
+            String currentWord = words.get(i).toString();
+            for (char ch : currentWord.toCharArray()) {
+                if (!charList.contains(ch)) {
+                    charList.add(ch);
+                }
+            }
+
+            String newWord = charList.toString();
+            newWord = newWord.replaceAll(",", "");
+            charList.clear();
+            uniqueList.add(i, newWord);
+        }
+        return uniqueList;
+    }
+
+    public static ArrayList findTwoWordsWithDuoLetters(ArrayList words) {
+        Map<Character, Integer> tempChar = new HashMap<>();
+        ArrayList twoWords = new ArrayList<>();
+        boolean ifTrue = false;
+
+        for (int i = 0; i < words.size(); i++) {
+            String currentWord = words.get(i).toString();
+            for (char ch : currentWord.toCharArray()) {
+                int count = tempChar.containsKey(ch) ? tempChar.get(ch) : 0;
+                tempChar.put(ch, count + 1);
+            }
+
+            for (Map.Entry<Character, Integer> pair : tempChar.entrySet()) {
+                if (pair.getValue() % 2 == 0) {
+                    ifTrue = true;
```

Comment:
Wrong logic here. It will work for any word which contains even number of one or more characters. But task says that we need words where even number of all characters